js/blocks/GraphicsBlocks.js

```diff
@@ -766,26 +778,30 @@ function setupGraphicsBlocks(activity) {
         flow(args, logo, turtle, blk) {
             const tur = activity.turtles.ithTurtle(activity.turtles.companionTurtle(turtle));
 
-            if (args.length === 1 && args[0].toString().length < 6) {
-                if (typeof args[0] === "string") {
-                    activity.errorMsg(NANERRORMSG, blk);
-                } else if (logo.inMatrix) {
-                    logo.phraseMaker.addRowBlock(blk);
-                    if (logo.pitchBlocks.indexOf(blk) === -1) {
-                        logo.pitchBlocks.push(blk);
-                    }
-                    logo.phraseMaker.rowLabels.push(activity.blocks.blockList[blk].name);
-                    logo.phraseMaker.rowArgs.push(args[0]);
-                } else if (tur.singer.inNoteBlock.length > 0) {
-                    tur.singer.embeddedGraphics[last(tur.singer.inNoteBlock)].push(blk);
+            if (args.length === 1) {
+                if (args[0] > 5000) {
+                   activity.errorMsg("Forward block value should be less than 5000.", blk); 
```

Comment:
same.

Reply:
We need to check for < -5000 too.

Reply:
@walterbender, should we update the wording to 'Value must be within the range of -5000 to 5000,' as 'Value should not exceed 5000' doesn't cover the cases where the value is less than -5000?

Reply:
Sounds good